cluster-up/cluster/kind-k8s-sriov-1.14.2/config_sriov.sh

```diff
@@ -37,6 +38,23 @@ function deploy_sriov_operator {
   popd
 }
 
+function deploy_network_resource_injector {
+  WEBHOOK_PATH=${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/network-resources-injector-${INJECTOR_GIT_HASH}
+  if [[ ! -d $WEBHOOK_PATH ]]; then
+    curl -L https://github.com/intel/network-resources-injector/archive/${INJECTOR_GIT_HASH}/network-resources-injector.tar.gz | tar xz -C ${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/
+  fi
+
+  pushd $WEBHOOK_PATH
+    make image
+    docker tag network-resources-injector localhost:5000/network-resources-injector
+    sed -i 's/network-resources-injector:latest/registry:5000\/network-resources-injector:latest/g' ./deployments/server.yaml
```

Comment:
You can do `s#ORIGINAL/PATH#NEW/PATH#` (`#` to avoid escaping of `/`). You can also drop `g` since there won't be multiple occurrences of the substring on a line.